consumer/aiml/guides/README.md

```diff
@@ -5,7 +5,8 @@ permalink: /documentation/consumer/aiml/guides/
 # Useful Guides
 
 These guides will help to get you started with a variety of available on-boards software
+- Mainline kernel Build Guide - [View](./build/mainline.md)(.md)
```

Comment:
I don't think we need the (.md) since all files are rendered to html. Just remove the '(.md)' and it should work fine